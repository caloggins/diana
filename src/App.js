import React from 'react'
import './App.css'
import { Container } from '@material-ui/core'
import QueuesPage from './queues/Page'

function App() {
    return (
        <div className="App">
            <Container maxWidth="md">
                <QueuesPage />
            </Container>
        </div>
    )
}

export default App
