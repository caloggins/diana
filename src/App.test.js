import React from 'react'
import { render } from '@testing-library/react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import App from './App'

describe('<App />', () => {
    xtest('renders learn react link', () => {
        const { getByText } = render(<App />)
        const linkElement = getByText(/learn react/i)
        expect(linkElement).toBeInTheDocument()
    })

    test('snapshot', () => {
        const tree = shallow(<App />)
        expect(toJson(tree)).toMatchSnapshot()
    })
})
