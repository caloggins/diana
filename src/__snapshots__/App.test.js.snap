// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> snapshot 1`] = `
<div
  className="App"
>
  <WithStyles(ForwardRef(Container))
    maxWidth="md"
  >
    <Page />
  </WithStyles(ForwardRef(Container))>
</div>
`;
